src/app/beer_garden/router.py

```diff
@@ -154,127 +183,182 @@ def forward(operation: Operation):
         RoutingRequestException: Could not determine a route to child
         UnknownGardenException: The specified target garden is unknown
     """
-    target_garden = None
-    for garden in gardens:
-        if garden.name == operation.target_garden_name:
-            target_garden = garden
-            break
-
-    if not target_garden:
+    try:
+        conn_type, conn_info = garden_connections[operation.target_garden_name]
+    except KeyError:
         raise UnknownGardenException(
             f"Unknown child garden {operation.target_garden_name}"
         )
 
-    if target_garden.connection_type is None:
-        target_garden = beer_garden.garden.get_garden(target_garden.name)
-        cache_gardens()
+    try:
+        if conn_type.casefold() == "http":
+            return _forward_http(operation, conn_info)
+        else:
+            raise RoutingRequestException(f"Unknown connection type {conn_type}")
```

Comment:
What about `https`?

Reply:
Consolidating this down to just http because the actual http vs https decision will be determined by the ssl boolean flag.

Reply:
I made changes in the UI to reflect this as well.